### PR TITLE
Update UG with assign member to task, assign deadline to project&task

### PR DESCRIPTION
**Pull request by modembcc**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -133,14 +136,72 @@ Deletes the specified project from the project list.
...
+- Failed assignment (example): `Person <PERSON_NAME> is not a team member.`
+
+### Assign deadline to project : `add deadline`
+
+Assigns a deadline to a project.
+
+Format: `add deadline <DEADLINE> /to <PROJECT_NAME>`
+
+- The specified project name must exist in the project list.
+- The deadline must be in `MMM D YYYY` format
```
Comment (Lin-Shuang-Shuang) [inline docs/UserGuide.md:176]: Should it be 'MM-DD-YYYY'?

**Review by Lin-Shuang-Shuang — CHANGES_REQUESTED** [review]
Apart from the date format, LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (modembcc) [reply docs/UserGuide.md:176]: I think including '-' would make the date format abmiguous.
MMM D YYYY is also a widely known format and should be fine.

**Review by Lin-Shuang-Shuang — APPROVED** [review]
LGTM

**Pull request merged**